hr_expense_sequence/README.rst

```diff
@@ -0,0 +1,90 @@
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/12.0-mig-hr_expense_sequence/hr_expense_sequence
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-12-0-mig-hr_expense_sequence/hr-12-0-mig-hr_expense_sequence-hr_expense_sequence
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/12.0-mig-hr_expense_sequence
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+You can change the default sequence (EX0001) by the one of your choice
+going to *Settings > Technical > Sequences & Identifiers > Sequences*, and
+editing the record `Expense Report sequence`.
+
+You will only have access to that section if your section has `Technical features`
+permission check marked.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/hr/issues/new?body=module:%20hr_expense_sequence%0Aversion:%2012.0-mig-hr_expense_sequence%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Serv. Tecnol. Avanzados - Pedro M. Baeza
+
+Contributors
+~~~~~~~~~~~~
+
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Pimolnat Suntian <pimolnats@ecosoft.co.th>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/hr <https://github.com/OCA/hr/tree/12.0-mig-hr_expense_sequence/hr_expense_sequence>`_ project on GitHub.
```

Comment:
URL is linked to wrong location, it should be,
https://github.com/OCA/hr/tree/12.0/hr_expense_sequence

Actually, you don't need to create README.rst file. From what I understand, the README.rst will be created by Bot automatically. Just make sure you have the good content in readme folder.